### PR TITLE
Added C-HR Hybird ( Lateral Only ) 

### PR DESCRIPTION
**Pull request by oivheg**
After tested your latest chages and using the modified firmware on the Panda , 
I am now able to use  my CHR Hybird, wihtout warning ( Lateral Only) .

### CODE REVIEW

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -125,8 +130,9 @@ def check_ecu_msgs(fingerprint, ecu):
   CAR.COROLLA: dbc_dict('toyota_corolla_2017_pt_generated', 'toyota_prius_2017_adas'),
   CAR.LEXUS_RXH: dbc_dict('lexus_rx_hybrid_2017_pt_generated', 'toyota_prius_2017_adas'),
   CAR.CHR: dbc_dict('toyota_chr_2018_pt_generated', 'toyota_prius_2017_adas'),
+  CAR.CHRH: dbc_dict('toyota_chr_hybrid_2017_pt_generated', 'toyota_prius_2017_adas'),
```
Comment (rbiasini) [inline selfdrive/car/toyota/values.py]: @oivheg you also need to add the generated dbc. I've already made the change directly in the opendbc repo. Can you copy it and update this PR?

File: selfdrive/car/toyota/values.py [hunk 2]
```diff
@@ -7,6 +7,7 @@ class CAR:
   COROLLA = "TOYOTA COROLLA 2017"
   LEXUS_RXH = "LEXUS RX HYBRID 2017"
   CHR = "TOYOTA C-HR 2018"
+  CHRH = "TOYOTA C-HR 2017 HYBRID"
```
Comment (rbiasini) [inline selfdrive/car/toyota/values.py]: HYBRID should be before the year

File: selfdrive/car/toyota/values.py (hunk 1 above)
Comment (oivheg) [reply selfdrive/car/toyota/values.py]: @rbiasini
 Thanks, I will do that .
I see that the file in opendbc is named "2018" and not 2017.
 I believe that it should be  changed to 2017, if I am not mistaken.

Comment (rbiasini) [reply selfdrive/car/toyota/values.py]: it's ok if it says 2018, I think it shipped one year sooner in EU

**Comment by rbiasini** [comment]
@oivheg the generated dbc file says 2018, while the source still says 2017. You should use the script  generator.py to autogenerate the "generated" one.

**Comment by oivheg** [comment]
Forogt that one.. 
it is now changed with the correct one .

**Pull request merged**